packages/vscode-docker-registries/src/clients/RegistryV2/RegistryV2DataProvider.ts

```diff
@@ -162,7 +162,8 @@ export abstract class RegistryV2DataProvider extends CommonRegistryDataProvider
             requestUri: requestUrl,
             scopes: [`repository:${item.parent.label}:pull`],
             headers: {
-                'accept': 'application/vnd.docker.distribution.manifest.v2+json,application/vnd.oci.image.index.v1+json'
+                // https://github.com/containerd/containerd/blob/main/core/remotes/docker/resolver.go#L163

```

Comment:
```suggestion
                // https://github.com/containerd/containerd/blob/c17839bdc7bf8699badf796078afc91fd54de6ac/core/remotes/docker/resolver.go#L163-L168
```

I'd suggest a permalink (one that links to a specific commit), so that if there are changes in the source file it doesn't cause the link in the comment to drift out of position